src/modules/Client/Service.php

```diff
@@ -100,7 +100,7 @@ public static function onAfterClientSignUp(\Box_Event $event)
             $email = [];
             $email['to_client'] = $params['id'];
             $email['code'] = 'mod_client_signup';
-            $email['password'] = $params['password'];
+            $email['password'] = 'The password you choose when creating your account.';
```

Comment:
String should be able to get  translated

Reply:
@Raul-Tech-Support The translation function we have is defined as `__trans`.
Example: `__trans('This is an example message');`

Reply:
@BelleNottelling I'm not completely sure how to implement this, I thought it would be:
`$email['password'] = __trans('The password you chose when creating your account.');`

However, that doesn't seem to work when I try it on my local version.

Reply:
Translation string might not exists in that case you should still see "The password you chose when creating your account." 

When we add it to our locale for next release after it has been merged it should work...

Reply:
@Raul-Tech-Support You have it correct in the code you sent, As Jaap said, the string isn't in the translation files yet, so it'll display the original string. But that's fine, we can add it to the locale after the fact, as long as the `__trans` function is used